src/EFCore.PG/Infrastructure/Internal/NpgsqlOptionsExtension.cs

```diff
@@ -196,29 +200,52 @@ public virtual NpgsqlOptionsExtension WithRemoteCertificateValidationCallback([C
 
     public class UserRangeDefinition
     {
-        /// <summary>The name of the PostgreSQL range type to be mapped.</summary>
+        /// <summary>
+        /// The name of the PostgreSQL range type to be mapped.
+        /// </summary>
+        [NotNull]
         public string RangeName { get; }
+
+        /// <summary>
+        /// The PostgreSQL schema in which the range is defined. If null, the default schema is assumed.
```

Comment:
Yeah, that's more or less what I meant, although we could provide more detail, up to you:

```suggestion
        /// The PostgreSQL schema in which the range is defined. If null, the default schema is used
        /// (which is public unless changed on the model).
```